### DISCUSSION TITLE
[Simulation question]: Issue with Cell Culture simulation

### DISCUSSION THREAD

**Discussion by 31Y5** [topas-nbio/TOPAS-nBio#40] [Simulation-question]
### Simulation question

Hello. I am having issues creating cell cultures that do not have overlapping cells no matter how big I make the world and culture. I have narrowed the problem down to a secret cell that will automatically overlap with my second cell. When I try to simulate 2 cells, it will generate 3 cells, where the 3rd is nested in the second for some reason. Is anyone else experiencing this issue? How should I go about fixing this?
See attached documents for geometry parameters and visualization.
![CellCultureVisualization](https://github.com/user-attachments/assets/24164490-d12f-43ac-a237-8f272e8ec1c9)
![GeometryParameter](https://github.com/user-attachments/assets/550d384e-6024-47f3-84f1-4a62a67721f1)

### TOPAS-nBio category

Geometry

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment]
Please test a small fix added to the main repository.
